crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -31,6 +31,9 @@ pub struct Create {
     #[clap(short, long)]
     pub add_profile: bool,
     // TODO (#253): think about supporting different account providers
+    /// Custom open zeppelin contract class hash of declared contract
+    #[clap(short, long)]
+    pub class_hash: Option<String>,
```

Comment:
```suggestion
    pub class_hash: Option<FieldElement>,
```
?

Reply:
Thought about it but wanted to adjust to current hashes types (that are strings)
Maybe one day we will change all hashes to be FieldElements :)